docusaurus/docs-internals/editor-controls/group-controls/tabs.md

```diff
@@ -15,23 +15,24 @@ Example of the `tabs` with position `left`:
+| `selector?` | `string`         | - | The CSS selector to which the styles will be applied                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
`tabs` doesn't have `selector` properties, so remove this and don't forget to remove also examples with `selector`